FiligranHQ/xtm-hub
# [front] -  Description on 1 line + tooltip

# Context 
UI Improvement in the Partner Vault 

The description of a document is truncated with '...' and the rest of the description is in a tooltip. 

# Tests on local : 
![image](https://github.com/user-attachments/assets/445eff94-e39b-4233-a282-115a791492cd)
 
# How to test : 
As an Admin Vault, add a document with a long description. 
Connect as a Partner Vault's user.
See the document with a long description. The description should be truncated with '...' 
Hover the description, a tooltip should appear with the full description.   

![image](https://github.com/user-attachments/assets/561798ca-7bc2-40b5-9779-a570e7dbe8e5)

# Close issue : 
close  #60
